changelog.md

```diff
@@ -1,4 +1,15 @@
 ## Changes to 1.8.xxx
+## Changes from 1.5.13 to 1.5.14
```

Comment:
Hm, this is kind of odd. How should we name it?

Reply:
This `## Changes from 1.5.13 to 1.5.14` section has to go down the changelog (where the 1.5.xxx changes live). And then you'll have to copy-paste it here (not the fixed bugs) too? And then the same spiel for 1.6 and 1.7

Reply:
Should I just add it to `Changes to 1.8.xxx`?

Reply:
Well, we need to show, that these changes (e.g. new command argument) are not only 1.8.xxx specific but also 1.7, 1.6 etc.

Reply:
That's kind of an odd situation. How are we doing that with backports. Should I also add the changes to each section?

Reply:
I took a look at the [1.7 changelog.md](https://github.com/mesosphere/marathon/blob/releases/1.7/changelog.md#changes-to-17xxx) it includes changes that we do not have on the `master` changelog. So I'm just going to copy it each time.

Reply:
> So I'm just going to copy it each time.

I think this is a better solution. People might only look at `releases/1.7/changelog.md`.